packages/pyxis-scss/src/scss/components/accordion.scss

```
@@ -0,0 +1,146 @@
+  display: grid;
+  grid-auto-flow: row;
+}
+
+.accordion-item {
+  @include foundations.radius(m);
+  background-color: foundations.color(neutral95);
+}
+
+.accordion-item--light {
+  background-color: foundations.color(neutralLight);
+}
+
+.accordion-item--alt {
+  background-color: rgba(foundations.color(neutralLight), 0.15);
+}
+
+.accordion-item__header {
+  @include foundations.columnGap(s);
+  @include foundations.radius(m);
+  @include foundations.horizontalPadding(s);
+  @include foundations.verticalPadding(xs);
+  @include mixins.focus(foundations.color(actionBase));
+  @include setColorAlt;
+  align-items: center;
+  border: lib.rem(2px) solid transparent;
+  color: foundations.color(neutralBase);
+  cursor: pointer;
+  display: flex;
+  transition: border-color foundations.duration(fast) foundations.timing(base);
+  width: 100%;
+
+  @include foundations.mq(xsmall) {
+    @include foundations.verticalPadding(s);
+    padding-left: foundations.spacing(l);
+    padding-right: foundations.spacing(l);
+  }
+
+  &:hover {
+    border-color: foundations.color(neutral85);
+    color: foundations.color(actionBase);
+  }
+
+  .accordion-item--alt & {
+    @include mixins.focus(foundations.color(neutralLight));
+
+    &:hover {
+      border-color: rgba(foundations.color(neutralLight), 0.15);
+      color: foundations.color(neutralLight);
+    }
+  }
+}
+
+.accordion-item__header__addon {
+  @include setColorAlt;
+  color: foundations.color(brandBase);
+  flex: fit-content 0 0;
+
+  @include foundations.mq(base, xsmall) {
+    max-width: lib.rem(48px);
+  }
+}
+
+.accordion-item__header__content-wrapper {
+  flex: auto;
+  text-align: left;
+  user-select: none;
+}
+
+.accordion-item__header__title {
+  @include foundations.text(l, bold);
+}
+
+.accordion-item__header__subtitle {
+  @include foundations.text(m, book);
+}
+
+.accordion-item__header__action-wrapper {
+  @include foundations.columnGap(3xs);
+  @include foundations.text(m, bold);
+  @include setColorAlt;
+  align-items: center;
+  color: foundations.color(actionBase);
+  display: grid;
+  flex: fit-content 0 0;
+  grid-auto-flow: column;
+}
+
+.accordion-item__header__action-icon {
+  border-radius: 50%;
+  transition:
+    transform foundations.duration(fast) foundations.timing(base),
+    background-color foundations.duration(fast) foundations.timing(base);
+
+  .accordion-item__header:hover & {
+    background-color: foundations.color(neutralLight);
+
+    .accordion-item--light & {
+      background-color: foundations.color(actionLight);
+    }
+
+    .accordion-item--alt & {
+      background-color: rgba(foundations.color(neutralLight), 0.15);
+    }
+  }
+}
+
+.accordion-item__header__action-icon--open {
+  transform: rotate(180deg);
+}
+
+.accordion-item__panel {
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height foundations.duration(fast) foundations.timing(base);
+  visibility: hidden;
+}
+
+.accordion-item__panel--open {
+  visibility: visible;
```

aggiungiamo anche una max-height, così se per un qualsiasi motivo non abbiamo l'altezza dell'accordion comunque si apre
```suggestion
  visibility: visible;
  max-height: fit-content;
 
```